README.rst

```diff
@@ -148,6 +148,7 @@ provided by each of the following software packages for your operating system.
   amsmath, amsfonts, amssymb, xcolor, authblk, geometry, caption, natbib,
   fancyhdr, lastpage, titlesec, enumitem, bibtex
 - Git_ (optional)
+- MATLAB Version 7.9 (R2009b) and Simulink Toolbox Version 7.4
```

Comment:
One note about this too. Can you print which toolboxes you need for the simulink stuff. I was not able to run the simulink stuff because it uses some blocks from other toolboxes that I do not have. Same goes for your m-files if you use any toolbox features.